get-screenshots/action.yaml

```diff
@@ -0,0 +1,130 @@
+name: Get screenshots
+description: Install a snap in a VM and get screenshots of it running
+author: Snapcrafters
+branding:
+  icon: camera
+  color: orange
+
+inputs:
+  issue-number:
+    description: "The issue number to post the screenshots into"
+    required: true
+  ci-repo:
+    description: "The repo to fetch tools/templates from. Only for debugging."
+    default: "snapcrafters/ci"
+    required: false
+  channel:
+    description: "The channel to publish the snap to"
+    default: "candidate"
+    required: false
+  github-token:
+    description: "A token with permissions to comment on issues"
+    required: true
+  screenshots-repo:
+    description: "The repository where screenshots should be uploaded."
+    default: "snapcrafters/ci-screenshots"
+    required: false
+  screenshots-token:
+    description: "A token with permissions to commit files to the screenshots repo"
+    required: true
+
+outputs:
+  screen:
+    description: "URL to a screenshot of the full screen of the VM"
+    value: ${{ steps.screenshots.outputs.screen }}
+  window:
+    description: "URL to a screenshot of the full screen of the VM"
+    value: ${{ steps.screenshots.outputs.window }}
+
+runs:
+  using: composite
+  steps:
+    - name: Checkout the code
+      uses: actions/checkout@v4
+
+    - name: Setup ghvmctl
+      uses: jnsgruk/ghvmctl/setup-ghvmctl@candidate
```

Comment:
I wonder if it makes sense to eventually bring this into snapcrafters too.

Reply:
More than happy to do so - I just put it under my name so I could iterate on it quickly during testing.

If you wanna setup a repo I'm happy to relocate it :)

Reply:
I created https://github.com/snapcrafters/ghvmctl

Currently without branch protection, so you can directly push. I'll add branch protections when the dust settles and everything else has landed.